backend/geonature/core/imports/checks/sql/utils.py

```diff
@@ -53,9 +53,7 @@ def report_erroneous_rows(imprt, entity, error_type, error_column, whereclause):
         cte = (
             update(transient_table)
             .values(
-                {
-                    transient_table.c[entity.validity_column]: False,
-                }
+                {transient_table.c[entity.validity_column]: False} if entity else {},
```

Comment:
Pourquoi ce changement ? Si une erreur est de niveau `ERROR`, une entité doit être fourni afin que la ligne soit marqué comme invalide !
Si tu as eu un problème avec cela, il y a peut-être un bug à investiguer, mais ceci n’est à priori pas la solution.

Reply:
Il y a des cas où l'entité peut être à None.
Il me semble que le comportement que tu décris: `niveau ERROR === entité fournie` n'est pas explicite. 
J'ai ajouté cette ligne pour ajouter un semblant de gestion de ce cas. Je suis ouvert à d'autres manière (error, exception, ignore, etc.).
Je trouve qu'un commentaire sur la ligne appelante est nettement insuffisant.
![image](https://github.com/PnX-SI/GeoNature/assets/150020787/4d3c2970-74fc-4012-bad2-f3a512a19e6d)

Reply:
Est-ce que tu as identifié un autre cas que lorsque l’erreur est de level `WARNING` ?
Sinon Ok of coures pour rajouter des commentaires dans `report_erroneous_rows` ainsi qu’éventuellement `assert entity is not None` dans le block `if …level == "ERROR"`